### PR TITLE
Fix auto-links and typos in coding-1

### PR DESCRIPTION
**Pull request by Katrist**
## Changes

Added missing auto-links and fixed several typos.

## Checks

By submitting your pull request for review, you agree to the following:

- [X] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [X] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [X] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

File: content/en-us/tutorials/fundamentals/coding-1/object-properties.md [hunk 1]
```diff
@@ -100,4 +100,4 @@ For the part, the script will change its Color property to a new Color3, a data
 
 All objects have properties. Parts have properties like color and transparency. At the same time, other object types have their unique properties.
 
-To change the color of a part, you need to be able to describe where to find it. If the part is in Workspace, use the keyword `workspace`. Then use dot operators to state the desired part and access its properties.
+To change the color of a part, you need to be able to describe where to find it. If the part is in Workspace, use the keyword `Class.Workspace|workspace`. Then use dot operators to state the desired part and access its properties.
```
Comment (aetter) [inline content/en-us/tutorials/fundamentals/coding-1/object-properties.md:103]:
```suggestion
To change the color of a part, you need to be able to describe where to find it. If the part is in Workspace, use the keyword `workspace`. Then use dot operators to state the desired part and access its properties.
```

**Review by aetter — APPROVED** [review]
I think you got a bit overzealous on this one. Since it's an introductory tutorial for people new to coding, there's something to be said for not linking too often and overwhelming them with reference materials. Likewise, for the things that are explicitly "type X," I don't think we want to rely on the autolink text.

**Review by aetter — APPROVED** [review]

**Comment by Katrist** [comment]
Understood, just wanted to be consistent with other similar documents.

**Comment by aetter** [comment]
> Understood, just wanted to be consistent with other similar documents.

Yeah, totally understood. I have the "consistency vs. ..." internal debate with myself all the time. 😄

**Pull request merged**